### ISSUE TITLE
Stop using deprecated pkg_resources

### ISSUE THREAD

**Issue by tfardet** [InseeFrLab/pynsee#219]
We should switch to the new imports described [here](https://setuptools.pypa.io/en/latest/pkg_resources.html), which would also avoid the setuptools dependency currently missing in ``requirements.txt``

**Comment by hadrilec** [comment]
issue tackled by the following commits: 1a0beb1c16c1b4005537c80644c19061bfa4ac94 526e918

**Comment by hadrilec** [comment]
is this issue still relevant?

**Comment by tfardet** [comment]
Yes, there are still a bunch of imports for ``pkg_resources`` throughout the code that need removing
